packages/client/hmi-client/src/utils/drag-drop.ts

```diff
@@ -0,0 +1,12 @@
+import { ref } from 'vue';
```

Comment:
Adding a commnet explaining the purpose of this composable.
Add MDN links to explain to the Map.